README.md

```diff
@@ -55,62 +157,90 @@ rails assets:precompile
 rails server
 ```
 
-
-**Run tests**
-
-1. To run all tests in the project
+5. Run server
+To run all tests in the project
 ```ruby
 rspec
 ``` 
-If you wanna run a certain test file
-```ruby
-rspec spec/test_file_path
-```
 
-## :blue_book: Learning Objectives
 
-- Use ruby gems as software packages system.
-- Install Ruby on Rails framework.
-- Understand Rails RESTful design and router.
-- Use controllers to handle requests and render empty views.
-- Use params from browser request in a safe way.
-- Use preprocessed html file with embedded Ruby code.
-- Use layouts and templates for shared content.
-- Use database migration files to maintain database schema.
-- Use validations for models.
-- Secure app from n+1 problems.
-- Understand what ORM is.
-- Write SQL queries with ActiveRecord.
-- Set up associations between models.
-- Build a webapp that requires the user to log in.
-- Use devise gem for authentication.
-- Limit access to webapp resources based on authorization rules.
-- Analyze in writing why you have made a coding choice using one structure over another.
 
-## Authors
 
-👤 **NeerajBhardwa**
 
-- GitHub: [@NeerajBhardwaj](https://github.com/rebel216)
-- Twitter: [@NeerajBhardwaj](https://twitter.com/rebel216)
-- LinkedIn: [NeerajBhardwaj](https://www.linkedin.com/in/neerajbhardwaj216/)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Neeraj Bhardwaj**
+
+- GitHub: [@Neeraj-Bhardwaj](https://github.com/rebel216)
+- Twitter: [@Neeraj-Bhardwaj](https://twitter.com/rebel216)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/neerajbhardwaj216/)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
 
-## 🤝 Contributing
+## 🔭 Future Features <a name="future-features"></a>
+
+- Live Demo
+- For desktop/Webpage
+- Expand on scale
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](https://github.com/rebel216/budget-app/issues).
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+I would like to thank...
```

Comment:
- [ ] The README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).